src/Symfony/Component/Messenger/Tests/Middleware/HandleMessageMiddlewareTest.php

```diff
@@ -335,6 +360,14 @@ public function __invoke()
     }
 }
 
+class FailingHandleMessageMiddlewareTestCallable
```

Comment:
Using an anonymous class in the test causes the array key to be a binary string - is this a problem?

Reply:
I thought I had an exception related to this but just tried now and all's good.